Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -255,14 +255,48 @@ function Upload-FilesToAzureContainer
             $blobStorageURI = $blobStorageEndpoint+$containerName+"/"+$blobPrefix
         }
 
+        $azcopy = "AzCopy\AzCopy.exe"
```

Comment:
As discussed let's skip this part of the change to move to Invoke-Tool since we have plans to change this task.